FileIO/VtkIO/VtuInterface.h

```diff
@@ -38,13 +38,20 @@ class VtuInterface
 	~VtuInterface();
 
 	/// Read an unstructured grid from a VTU file
-	/// \returns The converted mesh or a nullptr if reading failed
+	/// \return The converted mesh or a nullptr if reading failed
 	static MeshLib::Mesh* readVTUFile(std::string const &file_name);
 
 	/// Writes the given mesh to file.
-	/// \returns True on success, false on error
+	/// Note: a MPI_Comm type argument is needed.
```

Comment:
Just removed. Since there is MPI communicator used inside this function, this is just a reminder for future development if other communicator is employed. Actually there is already a comment in the definition of this function as a reminder.